README.md

```diff
@@ -1,4 +1,6 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/GannettDigital/jsonparser)](https://goreportcard.com/report/github.com/buger/jsonparser) ![License](https://img.shields.io/dub/l/vibe-d.svg)
+# Gannett Fork
+This is a fork of the jsonparser library from https://github.com/buger/jsonparser. We have established our own fork to fix a bug in which byte slices were being passed by reference then using an append which in turn modified the original data. A copy was added and used in the append. This solves the issue that can be seen https://github.com/buger/jsonparser/issues/145.
```

Comment:
i forgot but i also fixed one other bug here: https://github.com/buger/jsonparser/issues/164
Might be worth stating that the original repo does not appear to be maintained any longer and we want to be able to fix any bugs that pop up in the future